python/ray/air/util/tensor_extensions/arrow.py

```diff
@@ -62,6 +75,27 @@ def __arrow_ext_class__(self):
         """
         return ArrowTensorArray
 
+    if (
```

Comment:
How about making this a helper func (a centralized condition and comments that guard the control) since it's repeatedly used?

Reply:
Hmm yeah that seems doable, I'll make that change!